python/lsst/ts/standardscripts/enable_group.py

```diff
@@ -0,0 +1,87 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["EnableGroup"]
+
+import abc
+
+from lsst.ts import salobj
+
+
+class EnableGroup(salobj.BaseScript, metaclass=abc.ABCMeta):
+    """Base Script for enabling groups of CSCs.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    **Details**
+
+    """
+
+    __test__ = False  # stop pytest from warning that this is not a test
+
+    def __init__(self, index, descr):
+        super().__init__(index=index, descr=descr)
+
+        self.config = None
+
+    @property
+    @abc.abstractmethod
+    def group(self):
+        """Return group of CSC attribute.
```

Comment:
Could you please say more about this? What is a "group of CSC attribute"? What is the data type? I hope it's some existing class you can point to. I want to know what it should contain and how I make one. If it's an existing class I can look at the docs for that class.

Reply:
It should be a subclass of `lsst.ts.observatory.control.base_group.BaseGroup`... I'll document it here...